src/keys.rs

```diff
@@ -256,3 +256,63 @@ impl<C: CipherSuite> GroupVerifyingKey<C> {
         }
     }
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+    use crate::testing::Secp256k1Sha256;
+    use crate::utils::Scalar;
+    use crate::{FromBytes, ToBytes};
+    use ark_ff::UniformRand;
+    use rand::{rngs::OsRng, RngCore};
+
+    #[test]
+    fn test_serialization() {
+        let mut rng = OsRng;
```

Comment:
DQ: is it true that users should **not** use `OsRng` in production, not unless they are positive that their OS uses a proper CSPRNG? (That may not be the case on an embedded device for instance)

Reply:
Yeah for embedded devices you may have issues. I usually always use `OsRng` for testing as it's part of `rand_core`. Using `thread_rng` is good enough for seeding as well.